### ISSUE TITLE
Pickits question

### ISSUE THREAD

**Issue by Jakfox91** [kolton/d2bot-with-kolbot#869]
So i just started using this, and i can see i have two scripts automatically set up: kolton.nip and LLD.nip.
Do i need both? or should i remove one? I want my guy to pickup runes and good items in general. Or do i need to write the whole thing myself?

**Comment by aim2kill** [comment]
you need then unless you can make your own or find some you can use [https://github.com/kolton/d2bot-with-kolbot/issues/834#issuecomment-448981256](url)

**Comment by Jakfox91** [comment]
> you need then unless you can make your own or find some you can use [https://github.com/kolton/d2bot-with-kolbot/issues/834](url)

So i should keep the two scripts attached? They will do the job for me?

**Comment by aim2kill** [comment]
Ill suggest you to make 1 for unique,set,magic/rares,white(base), etc
This is the best I can do for you
https://github.com/aim2kill/copy/tree/master/d2bot-with-kolbot-master/d2bs/kolbot/pickit

**Comment by Jakfox91** [comment]
> Ill suggest you to make 1 for unique,set,magic/rares,white(base), etc
> This is the best I can do for you
> https://github.com/aim2kill/copy/tree/master/d2bot-with-kolbot-master/d2bs/kolbot/pickit

So, make one for each of them and then link it in the script? So the kolton.nip and LLD.nip doesnt cover it?

**Comment by aim2kill** [comment]
They cover alot to be honest but its not the best

**Comment by Jakfox91** [comment]
> t

Okay, so i made one for: charms, sets, uniques, whites and rares now. does that sound about right?

**Comment by aim2kill** [comment]
Yes sound good.
Do you mean run as the same time with kolton and LLD? If that the question I wont recomment that

**Comment by Jakfox91** [comment]
> Yes sound good.
> Do you mean run as the same time with kolton and LLD? If that the question I wont recomment that

yeah, exactly. okay, ill remove them then.

**Comment by aim2kill** [comment]
This site is you friend http://classic.battle.net/diablo2exp/items/normal/uarmor.shtml

**Comment by Jakfox91** [comment]
> This site is you friend http://classic.battle.net/diablo2exp/items/normal/uarmor.shtml

Right, but now that ive added the scripts you send me, i dont really need to do anything more, do i?
